deno.json

```diff
@@ -28,9 +28,9 @@
     "std/": "https://deno.land/std@0.188.0/",
     "stripe": "./stripe.ts",
     "feed": "https://esm.sh/feed@4.2.2",
-    "fresh_charts/": "https://deno.land/x/fresh_charts@0.2.2/",
     "kv_oauth": "https://deno.land/x/deno_kv_oauth@v0.2.5/mod.ts",
-    "@twind/core": "https://esm.sh/@twind/core@1.1.3"
+    "@twind/core": "https://esm.sh/@twind/core@1.1.3",
+    "chart.js": "https://esm.sh/chart.js@4.0.1/auto"
```

Comment:
intentionally chose to stick to `4.0.1` for now.  
on `4.3.0` some of the core plugins seem to be broken.  
also the rendering seems to be changed, as the canvas doesn't respect the container's with anymore.  
thus the charts would always be rendered blurry at 300x150 each.

Reply:
tried to bump to `4.3.0` again.
however the animations as well as the intersection for the tooltips didn't seem to work either.
i'd try to do some further investigations on why that happens.